nixnet/_enums.py

```diff
@@ -2029,3 +2029,26 @@ class FrameType(enum.Enum):
     SPECIAL_DELAY = _cconsts.NX_FRAME_TYPE_SPECIAL_DELAY
     SPECIAL_LOG_TRIGGER = _cconsts.NX_FRAME_TYPE_SPECIAL_LOG_TRIGGER
     SPECIAL_START_TRIGGER = _cconsts.NX_FRAME_TYPE_SPECIAL_START_TRIGGER
+
+class Terminals(enum.Enum):
+    "Terminals to import/export from the XNET device"
+    NX_TERM_FRONTPANEL0 = _cconsts.NX_TERM_FRONT_PANEL0
+    NX_TERM_FRONTPANEL1 = _cconsts.NX_TERM_FRONT_PANEL1
+    NXTERM_PXI_TRIG0 = _cconsts.NX_TERM_PXI_TRIG0        
+    NXTERM_PXI_TRIG1 = _cconsts.NX_TERM_PXI_TRIG1
+    NXTERM_PXI_TRIG2 = _cconsts.NX_TERM_PXI_TRIG2
+    NXTERM_PXI_TRIG3 = _cconsts.NX_TERM_PXI_TRIG3
+    NXTERM_PXI_TRIG4 = _cconsts.NX_TERM_PXI_TRIG4
+    NXTERM_PXI_TRIG5 = _cconsts.NX_TERM_PXI_TRIG5
+    NXTERM_PXI_TRIG6 = _cconsts.NX_TERM_PXI_TRIG6
+    NXTERM_PXI_TRIG7 = _cconsts.NX_TERM_PXI_TRIG7
+    NXTERM_FRONTPANEL0 = _cconsts.NX_TERM_FRONT_PANEL0      
+    NXTERM_FRONTPANEL1 = _cconsts.NX_TERM_FRONT_PANEL1    
+    NXTERM_PXI_STAR = _cconsts.NX_TERM_PXI_STAR
+    NXTERM_PXI_CLK10 = _cconsts.NX_TERM_PXI_CLK10
+    NXTERM_10MHZTIMEBASE = _cconsts.NX_TERM_10_M_HZ_TIMEBASE
+    NXTERM_1MHZTIMEBASE = _cconsts.NX_TERM_1_M_HZ_TIMEBASE
+    NXTERM_MASTERTIMEBASE = _cconsts.NX_TERM_MASTER_TIMEBASE
+    NXTERM_COMMTRIGGER = _cconsts.NX_TERM_COMM_TRIGGER
+    NXTERM_STARTTRIGGER = _cconsts.NX_TERM_START_TRIGGER
+    NXTERM_LOGTRIGGER = _cconsts.NX_TERM_LOG_TRIGGER
```

Comment:
How come the words were combined (`STARTTRIGGER` rather than `START_TRIGGER`)?

Reply:
Both the XNET manual and the equivalent constant in the C API have the words joined as one. I will separate them for readability.

Reply:
I think where the confusion is coming in is definition of "word".  In programming, there are several conventions for splitting words beyond " ". 
- Capital letters (camelCase, PascalCase)
- underscores (snake_case, SCREAMING_CASE)
- dashes (arrow-case).

So we're switching from PascalCase to SCREAMING_CASE, so we replace the capital letter split with an underscore is the appropriate thing to do.

When I bootstrapped the API, I had a script automatically change the convention for us.  It mostly worked (for example `nxTerm_LogTrigger` was correctly mapped to `NX_TERM_LOG_TRIGGER`).  In some cases, it couldn't quite tell what the word boundaries are and got it very wrong (`PDUs` -> `PD_Us` -> `pd_us`).  Other cases require some more thought on our part (should `MHz` be `M_HZ` or `MHZ`?).

For general reference, the constants look like
- `nxTerm_FlexRayStartCycle `
- `nxTerm_PXI_Star`
- `nxTerm_PXI_Trig4`

The odd thing is that the C API is sometimes splitting with `_`.  In most cases, it is used to separate phrases ("NIXNET Terminal" from "FlexRay Start Cycle") but "PXI" and "Star" are part of the same phrase.